### ISSUE TITLE
Syntax error in SQL syntax

### ISSUE THREAD

**Issue by dzschille** [magento-hackathon/EAVCleaner#21]
I get this error message:

```
    SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '* FROM catalog_eav_attribute WHERE `attribute_id ` not in(SELECT `attribute_id` F' at line 1
```

Fixed in #20.

**Comment by sprankhub** [comment]
This issue is fixed and merged in #20 and can be closed.

**Comment by dzschille** [comment]
Great, thanks!
